test/unit/containers/sound-editor.test.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react'; // eslint-disable-line no-unused-vars
 import {mountWithIntl} from '../../helpers/intl-helpers';
 import configureStore from 'redux-mock-store';
 import mockAudioBufferPlayer from '../../__mocks__/audio-buffer-player.js';
-
+import {IntlProvider as Intl} from 'react-intl'; // eslint-disable-line no-unused-vars
```

Comment:
Woops that was left over (and lint didn't catch it because it was force disabled!) good catch